app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt

```diff
@@ -106,25 +114,37 @@ private fun GroupConversationDetailsContent(
     val elevationState by remember { derivedStateOf { lazyListStates[currentTabState].topBarElevation(maxAppBarElevation) } }
 
     val sheetState = rememberModalBottomSheetState(ModalBottomSheetValue.Hidden)
-    val openBottomSheet: () -> Unit = remember { { scope.launch { sheetState.show() } } }
-    val closeBottomSheet: () -> Unit = remember { { scope.launch { sheetState.hide() } } }
+    val openBottomSheet: () -> Unit = remember { { scope.launch { sheetState.animateTo(ModalBottomSheetValue.Expanded) } } }
 
     val deleteGroupDialogState = rememberVisibilityState<GroupDialogState>()
     val leaveGroupDialogState = rememberVisibilityState<GroupDialogState>()
+    val clearConversationContentDialogState = rememberVisibilityState<DialogState>()
 
-    if(!isLoading) {
-        deleteGroupDialogState.dismiss()
-        leaveGroupDialogState.dismiss()
-    }
     WireModalSheetLayout(
         sheetState = sheetState,
         coroutineScope = rememberCoroutineScope(),
         sheetContent = {
-            ConversationGroupDetailsBottomSheet(
-                conversationOptionsState = groupOptionsState,
-                closeBottomSheet = closeBottomSheet,
-                onDeleteGroup = deleteGroupDialogState::show,
-                onLeaveGroup = leaveGroupDialogState::show
+            ConversationSheetContent(
+                conversationSheetState = rememberConversationSheetState(
+                    conversationSheetContent = ConversationSheetContent(
+                        title = groupOptionsState.groupName,
+                        conversationId = groupOptionsState.conversationId,
+                        mutingConversationState = groupOptionsState.mutedState,
+                        conversationTypeDetail = ConversationTypeDetail.Group(
+                            conversationId = groupOptionsState.conversationId,
+                            isCreator = groupOptionsState.isAbleToRemoveGroup
+                        )
+                    ),
+                    conversationOptionNavigation = ConversationOptionNavigation.Home
+                ),
+                onMutingConversationStatusChange = { },
+                addConversationToFavourites = { /*TODO*/ },
+                moveConversationToFolder = { /*TODO*/ },
+                moveConversationToArchive = { /*TODO*/ },
+                clearConversationContent = clearConversationContentDialogState::show,
+                blockUser = { _, _ -> },
+                leaveGroup = leaveGroupDialogState::show,
+                deleteGroup = deleteGroupDialogState::show,
```

Comment:
used already existing bottom sheet, no need to create another one as they have the same functionalities and same designs